### ISSUE TITLE
More issues in div-alg

### ISSUE THREAD

**Issue by angeldelriomateos** [gap-packages/wedderga#51]
I discover more issues in the functions in div-alg. Allen Herman is going to fix it. I wonder whether we should make this public. I guess it is not important if the bug is fixed quickly

**Comment by olexandr-konovalov** [comment]
@angeldelriomateos yes, the best way would be to describe them here, under this issue (or in several issues, if they can be individually addressed).

**Comment by olexandr-konovalov** [comment]
@angeldelriomateos if you want to describe these issues, you can do this here.

**Comment by olexandr-konovalov** [comment]
Not enough details provided here, and there were fixed in that file already. I am closing this, if there will be anything new, it should be reported in a new issue.
